datadog_checks_dev/datadog_checks/dev/tooling/snmp_profile_validator/snmp_profile.json

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
Let's move this in `datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp`

Reply:
Done!

Reply:
🤔 I would definitely keep it this file here...
IMHO the JSON schema has nothing to do with the "command/CLI" part of the validator, and having it here looks more aligned with how the `manifest_validator` and `config_validator` are organised...
Now, @ofek you have the last word on it :-)

Reply:
Questions:

1. I'm fine with moving it back but afaict only the snmp meta feature uses snmp stuff so imo makes more sense to have everything there
2. Is this file generated or manually constructed? If the latter, can we store it in Python instead as a `dict`?

Reply:
The json schema was generated and then edited manually. We would like to keep it as a json file in order to be able to use it in a validator extension for VSCode, or potentially to auto-generate documentation.